ta/pkcs11/src/token_capabilities.c

```diff
@@ -231,6 +232,7 @@ const struct pkcs11_mechachism_modes token_mechanism[] = {
 	TA_MECHANISM(PKCS11_CKM_AES_ECB, CKFM_CIPHER_WRAP),
 	TA_MECHANISM(PKCS11_CKM_AES_CBC, CKFM_CIPHER_WRAP),
 	TA_MECHANISM(PKCS11_CKM_AES_CTR, CKFM_CIPHER),
+	TA_MECHANISM(PKCS11_CKM_AES_GCM, CKFM_CIPHER),
```

Comment:
This line should be added at the end of the patch series, once AES GCM is supported by the pkcs11 TA.